components/python/salt/salt-PYVER.p5m

```diff
@@ -0,0 +1,1653 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2018 Till Wegmueller
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)-$(PYV)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value='Portable, distributed, remote execution and configuration management system'
+set name=info.classification value='org.opensolaris.category.2008:System/Administration and Configuration'
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='Apache License 2.0'
+
+# Salt does a lot of try/import/except, making pkgdepend unhapyy
+# If one disables this line pkg will try to add a giant list of optional dependencies to the package
+# Salt uses try import on purpose. Salt requires the admin to add optional dependencies via it's states. 
+<transform path=usr/lib/python$(PYVER)/vendor-packages/salt/.*\.py -> default pkg.depend.bypass-generate .* >
+
+# force a dependency on the Python runtime
+depend type=require fmri=__TBD pkg.debug.depend.file=python$(PYVER) \
+    pkg.debug.depend.path=usr/bin
+
+# dependencies
+depend type=require fmri=library/python/msgpack-python-$(PYV)
+depend type=require fmri=library/python/pyyaml-$(PYV)
+depend type=require fmri=library/python/jinja2-$(PYV)
+depend type=require fmri=library/python/markupsafe-$(PYV)
+depend type=require fmri=library/python/requests-$(PYV)
+depend type=require fmri=library/python/tornado-$(PYV)
+depend type=require fmri=library/python/pyzmq-$(PYV)
+depend type=require fmri=library/python/cryptography-$(PYV)
+depend type=require fmri=library/python/futures-$(PYV)
+
+# SMF Manifest and distribution directories
+set name=org.opensolaris.smf.fmri value=svc:/application/salt:minion value=svc:/application/salt:master
+dir  path=etc/salt owner=root group=root mode=0755
```

Comment:
Do we have some example configuration files?

Reply:
Not needed. But the Directory is needed to force salt to use it. We force that in the manifest with the -d switch. If we do not force the configuration directory salt does not use the same paths as documented leading to people being confused.